README.md

```diff
@@ -46,9 +46,8 @@ You can find the library in `window.logger`.
 ## Usage
 
 ```js
-app({
+logger()(app)({
```

Comment:
@okwolf Can you specify some options here too? or are there not many options?

Reply:
@JorgeBucaran there's only one option, and it's usage is shown below.